web/src/app/util/avatar/types.js

```diff
@@ -22,7 +22,7 @@ function useValidImage(srcURL) {
   return valid
 }
 
-function renderAvatar(Fallback, otherProps, imgSrc) {
```

Comment:
As this is a render function (and not a Component) shouldn't it stay as `renderAvatar`?

Reply:
Future context: the new rule wants this to either be a component or hook since it uses a hook itself; so we changed it to `useAvatar` to be named as a hook.